### DISCUSSION TITLE
Issue with python serverless

### DISCUSSION THREAD

**Discussion by productlicenses** [vercel/community#6557] [Help]
### Summary

I am seeing in the logs the following

/opt/wrapper: does not exist INIT_REPORT Init Duration: 2.50 ms Phase: init Status: error Error Type: Runtime.ExitError /opt/wrapper: does not exist INIT_REPORT Init Duration: 2.34 ms Phase: invoke Status: error Error Type: Runtime.ExitError Error: Runtime exited with error: exit status 127

This code previously worked one month ago, node is 18.x, using python serverless with Flask and an index.py

### Steps to Reproduce

Just try using any kind of python serverless function

**Comment by QuiiBz** [comment]
👋 Could you try to re-deploy your project now? The issue should be fixed.
